src/HornedBeasts.js

```diff
@@ -10,19 +10,25 @@ class HornedBeasts extends React.Component{
         }
     }
     
-    handleClick = () => {
-        this.setState({
-            favorites: this.state.favorites + 1
-        });
-    }
+    // handleClick = () => {
+    //     this.setState({
+    //         favorites: this.state.favorites + 1
+    //     });
+    //     this.props.displayModal();
+    // }
```

Comment:
My feedback here would be to remove this commented out code if we are done (looks like we moved this code to another component 👍  )